integration_test/third_party_apps_data/applications/activemq/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/activemq"
-short_name: activemq
+short_name: ActiveMQ
 long_name: Apache ActiveMQ
 description: -|
```

Comment:
Should this be `|-` like the other files?

Reply:
Short answer: No.
|- is for multiline string like long description. See more in: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines

Reply:
You are right... i thought you asked if short_name needs |-...🤦‍♀️